spec/sidekiq/evss/disability_compensation_form/submit_form0781_spec.rb

```diff
@@ -88,12 +88,53 @@
 
       it 'updates a StatsD counter and updates the status on an exhaustion event' do
         subject.within_sidekiq_retries_exhausted_block({ 'jid' => form526_job_status.job_id }) do
+          # Will receieve increment for failure mailer metric
+          allow(StatsD).to receive(:increment).with(
+            'shared.sidekiq.default.EVSS_DisabilityCompensationForm_Form0781DocumentUploadFailureEmail.enqueue'
+          )
```

Comment:
Does the `form526_send_0781_failure_notification` Flipper need to be enabled in this context?

Reply:
Nah, the behavior here is unaffected by the Flipper